installing/installing_ibm_z/installation-config-parameters-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-ibm-z-kvm"]
+= Installation configuration parameters for IBM Z(R) and {linuxoneProductName} with {op-system-base} KVM
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-ibm-z-kvm
+:platform: IBM Z
+
+toc::[]
+
+Before you deploy an {product-title} cluster on {ibmzProductName}, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
+
+[NOTE]
```

Comment:
@SNiemann15 Is this a holdover from when you were going to use a single assembly twice in the TOC. Given that we now have two separate assemblies, I am not sure if the note is required. You could always add it back when you deliver https://issues.redhat.com/browse/OSDOCS-7108